framework/src/transfers/MultiAppTransfer.C

```diff
@@ -33,7 +33,7 @@ validParams<MultiAppTransfer>()
   // MultiAppTransfers by default will execute with their associated MultiApp. These flags will be
   // added by FEProblemBase when the transfer is added.
   ExecFlagEnum & exec_enum = params.set<ExecFlagEnum>("execute_on", true);
-  exec_enum.addAvailableFlags(EXEC_SAME_AS_MULTIAPP);
+  exec_enum.addAvailableFlags(EXEC_SAME_AS_MULTIAPP, EXEC_FINAL);
```

Comment:
Wait is this needed? EXEC_FINAL should already be included in the Multiapp's flags? Maybe it's not, but we should probably solve this ticket that way anyway.

Reply:
I will clean this up as well.